sormas-backend/src/main/java/de/symeda/sormas/backend/campaign/CampaignFacadeEjb.java

```diff
@@ -222,6 +232,18 @@ private void validate(CampaignDto campaignDto) {
 							.getValidationError(Validations.campaignDashboardChartValueNull, CampaignDashboardElement.TAB_ID, campaignDto.getName()));
 				}
 
+				if (cde.getSubTabId() == null || cde.getSubTabId().isEmpty()) {
+					if (oneSubTabIsNotNullOrEmptyMap.get(cde.getTabId())) {
```

Comment:
The check doesn't work like that, when only the last sub tab id is not empty. It would go through the previous ones and be ok, because until then all sub tabs are empty and only for the last row the sub tab id would be added to the map.